src/hotspot/os/linux/os_linux.hpp

```diff
@@ -360,19 +369,24 @@ class os::Linux {
   }
   // Check if bound to only one numa node.
   // Returns true if bound to a single numa node, otherwise returns false.
-  static bool is_bound_to_single_node() {
+  static bool is_bound_to_single_mem_node() {
```

Comment:
```suggestion
  // Check if memory is bound to only one numa node.
  // Returns true if memory is bound to a single numa node, otherwise returns false.
  static bool is_bound_to_single_mem_node() {
```

Reply:
Done.